Clappr.xcodeproj/xcshareddata/xcschemes/Clappr_Tests.xcscheme

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
```

Comment:
I think you don't need to merge this file

Reply:
See comment above